pkg/server/server.go

```diff
@@ -35,6 +35,13 @@ func New(config configuration.Configuration, application application.Application
 	ginRouter.Use(
 		gin.LoggerWithWriter(gin.DefaultWriter, "/api/v1/health"),
 		gin.Recovery(),
+		cors.New(cors.Config{
+			AllowOrigins:     []string{"*"},
+			AllowMethods:     []string{"*"},
```

Comment:
Hm... "*" doesn't seem to be always treated as equivalent of all methods... https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Methods
I wonder if we should use 
```suggestion
			AllowMethods:     []string{"PUT", "PATCH", "POST", "GET", "DELETE", "OPTIONS"},
```
instead?